src/core/test/dom.test.ts

```diff
@@ -32,6 +32,7 @@ describe('DOM test utils', () => {
         <button>2</button>
         <button>2</button>
       </div>
+      <div class="second-type">should not match</div>
```

Comment:
Added an extra `.second-type` element outside `LIST_WITH_ITEMS_CLASS_NAME` to make sure it does not get matched by `findAny`